mediathread/templates/main/course_list.html

```diff
@@ -52,6 +50,11 @@
             <li class="{% if semester_view == 'future' %}active{% endif %}">
                 <a id="future_semesters" href="?semester_view=future">Future</a>
             </li>
+            <li class="navbar-right">
```

Comment:
Thanks, I added an `add_privilege` check just like the existing course list page uses.